sqlitex/exec.go

```diff
@@ -76,13 +78,14 @@ type ExecOptions struct {
 // query using the [sqlite.Stmt] Bind* methods. Basic reflection on args is used
 // to map:
 //
-//	integers to BindInt64
-//	floats   to BindFloat
-//	[]byte   to BindBytes
-//	string   to BindText
-//	bool     to BindBool
+//	integers    to BindInt64
+//	floats      to BindFloat
+//	[]byte      to BindBytes
+//	string      to BindText
+//	bool        to BindBool
+//      untyped nil to BindNull
 //
-// All other kinds are printed using fmt.Sprintf("%v", v) and passed
+// All other kinds are printed using fmt.Sprint(v) and passed
```

Comment:
nit: And use a [link](https://go.dev/doc/comment#doclinks) here:

```suggestion
// All other kinds are printed using [fmt.Sprint] and passed
```